modules/@apostrophecms/notification/index.js

```diff
@@ -56,15 +55,26 @@ module.exports = {
       throw self.apos.error('unimplemented');
     },
     patch(req, _id) {
-      throw self.apos.error('unimplemented');
+      return self.db.updateOne({ _id }, {
+        $set: {
+          ...req.body
```

Comment:
Must launder these patchable fields before patching them, without inadvertently creating them as properties of req.body if they are not present.

Reply:
Done